cmd/limactl/main.go

```diff
@@ -143,6 +143,11 @@ func newApp() *cobra.Command {
 		}
 
 		if cmd.Flags().Changed("yes") {
+			switch cmd.Name() {
+			case "clone", "edit":
+				logrus.Warn("--yes flag is deprecated for the '" + cmd.Name() + "' command, use --start instead and --tty=false")
```

Comment:
I don't really understand the warning. `--start` has different semantics than `--yes`, so `--start=false` may be the same as `--yes=true`, but `--start=true` (start without asking) is different from `--yes=false` (ask about starting).

I think the `edit` and `clone` commands should just print the simplified message:

```suggestion
				logrus.Warn("The --yes flag is deprecated")
```

Users can then run `limactl edit --help` to learn more.